src/main/kotlin/com/mtereshchuk/account/util/Result.kt

```diff
@@ -0,0 +1,32 @@
+package com.mtereshchuk.account.util
+
+import com.mtereshchuk.account.model.AccountId
+
+/**
+ * @author mtereshchuk
+ */
+sealed interface Result<T>
```

Comment:
In such situations, when neighbor classes interact with each other, I prefer to use the Go-like (Either-like) approach for handling errors to not raise and catch Exceptions in the same package, because it just wastes resources of JVM